doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -855,7 +855,10 @@ A subscription is represented by a JSON object with the following fields:
 + `id`: Subscription unique identifier. Automatically created at creation time.
 + `subject`: It is an object that describes the subject of the subscription.
 + `notification`: It is an object that describes the notification received by the subscriber.
-+ `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
++ `expires`: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit
+   this field.
```

Comment:
In that case, I would insist on clearly explaining that "expires" needs to be an exact date, that "P1M" is ISO8601, but the broker will not accept it.

Also, as discussed somewhere, the date must be complete with hour, minute and second (that *might* be part of ISO8601, but still, it is never a bad idea to explain it).

About "expires" vs "expirationDate" - "expires" gives me the feeling of a boolean. "expirationDate" on the other hand is very clear. It also removes the doubt about using "P1M", which is not a date, but a period of time.